utils.py

```diff
@@ -528,26 +558,23 @@ def check_libs(libraries):
         if have_lib_result:
             logging.info(f"* {library} is installed!")
         else:
-            if config.PACKAGE_MANAGER_COMMAND:
-                message = f"Your {config.OS_NAME} install is missing the library: {library}. To continue, the script will attempt to install the library by using {config.PACKAGE_MANAGER_COMMAND}. Proceed?"
-                if cli_continue_question(message, config.EXTRA_INFO):
+            if config.PACKAGE_MANAGER_COMMAND_INSTALL:
+                message = f"Your {config.OS_NAME} install is missing the library: {library}. To continue, the script will attempt to install the library by using {config.PACKAGE_MANAGER_COMMAND_INSTALL}. Proceed?"
+                if cli_continue_question(message, "", ""):
                     install_packages(config.PACKAGES)
             else:
-                logos_error(f"The script could not determine your {config.OS_NAME} install's package manager or it is unsupported. Your computer is missing the library: {library}. Please install the package associated with {library} for {config.OS_NAME}.\n{config.EXTRA_INFO}")
-
-def checkDependencies():
-    logging.info("Checking system's for dependencies:")
-    cmds = ["wget"]
-    check_commands(cmds)
-
-def checkDependenciesLogos10():
-    logging.info("All dependencies found. Continuing…")
-
-def checkDependenciesLogos9():
-    logging.info("Checking dependencies for Logos 9.")
-    cmds = ["xwd", "cabextract"]
-    check_commands(cmds);
-    logging.info("All dependencies found. Continuing…")
+                logos_error(f"The script could not determine your {config.OS_NAME} install's package manager or it is unsupported. Your computer is missing the library: {library}. Please install the package associated with {library} for {config.OS_NAME}.")
+
+def check_dependencies():
+    if int(config.TARGETVERSION) == 10:
+        logging.info("Checking Logos 10 dependencies")
+        install_dependencies(config.PACKAGES)
+    elif int(config.TARGETVERSION) == 9:
+        logging.info("Checking Logos 9 dependencies")
+        install_dependencies(config.PACKAGES)
+        install_dependencies(config.L9PACKAGES)
```

Comment:
I know of at least one person who has continued using Logos 9, and another who keeps trying to go to Logos 10 but reverts to Logos 9.

I would currently consider the Logos 9 code broken in this rewrite. We would need to test this more robustly. We may also need to reconsider how we determine what packages are needed so as to run an install of dependencies only once.